experiments/AMIP/modular/coupler_driver_modular.jl

```diff
@@ -238,11 +236,14 @@ if mode_name == "amip"
     SST_init = interpolate_midmonth_to_daily(date0, SST_info)
     ocean_sim = SurfaceStub((;
         T_sfc = SST_init,
+        ρ_sfc = ClimaCore.Fields.zeros(boundary_space),
         z0m = FT(1e-3),
         z0b = FT(1e-3),
         beta = FT(1),
         α = FT(0.06),
         area_fraction = (FT(1) .- land_fraction),
+        phase = TD.Liquid(),
```

Comment:
Why don't we need to specify the water phase in the land model? I know beta provides information about how saturated the surface is for evaporation, but do we need to describe whether its saturated with water vs ice vs both?

Reply:
It's already contained internally in Land. It allows for both `Liquid()` and `Ice()`, depending on the dynamic snow fraction.